Provider/src/FirebirdSql.Data.FirebirdClient/Client/Native/FesBlob.cs

```diff
@@ -160,7 +160,7 @@ protected override byte[] GetSegment()
 					}
 				}
 
-				return segment.ToArray();
+				segment.WriteTo(ms);
```

Comment:
Why not to write directly to `ms` above?

Reply:
That i did not notice, refactored in following commit